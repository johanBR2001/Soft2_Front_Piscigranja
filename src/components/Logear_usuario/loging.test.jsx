import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { MemoryRouter } from 'react-router-dom';

import Login from './login';

configure({ adapter: new Adapter() });

describe('Login Component', () => {
    it('should render the login form', () => {
        const wrapper = shallow(
            <MemoryRouter>
              <Login />
            </MemoryRouter>
          );
      expect(wrapper).toMatchSnapshot();
    });

    it('should handle email input change', () => {
        const wrapper = shallow(
          <MemoryRouter>
            <Login />
          </MemoryRouter>
        );
        const emailInput = wrapper.find('input[type="email"]');
        emailInput.simulate('change', { target: { value: 'test@example.com' } });
        expect(wrapper.state('email')).toEqual('test@example.com');
      });
});