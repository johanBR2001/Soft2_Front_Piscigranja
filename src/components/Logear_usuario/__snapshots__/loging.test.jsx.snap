// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component should render the login form 1`] = `ShallowWrapper {}`;
